website/src/pages/index.mdx

```diff
@@ -39,20 +39,25 @@ salaire brut: 2300 €/mois
 ## Qui utilise publicodes ?
 
 - **[Mon-entreprise](https://mon-entreprise.urssaf.fr/simulateurs)** utilise
-  publicodes pour spécifier l'ensemble des calculs relatifs à la législation
-  socio-fiscale en France. Le site permet entre autres de simuler une fiche de
-  paie complète, de calculer les cotisations sociales pour un indépendant ou
-  encore connaître le montant du chômage partiel.
-- **[Nos Gestes Climat](https://nosgestesclimat.fr)** propose un
-  calculateur d'empreinte climat personnel de référence complètement ouvert.
+  publicodes pour implémenter la législation socio-fiscale dans 
+  des simulateurs (paie, cotisations, impôts, droits ouverts)
+- **[Nos Gestes Climat](https://nosgestesclimat.fr)** propose le
+  calculateur d'empreinte climat personnelle de référence, complètement ouvert.
 - **[Code du Travail Numérique](https://code.travail.gouv.fr/outils/preavis-retraite)**
   développe un simulateur de préavis de retraite intégrant de nombreuses
   conventions collectives.
-- **[Mes Aides Vélos](https://mesaidesvelo.fr)** trouve les aides pour acheter
-  un vélo dans votre ville.
-- **[futur.eco](https://futur.eco/)** calcule les bilans carbone d'un grand
-  nombre d'activités, plats, transports ou biens.
-- **[Impact CO2](https://impactco2.fr/)** le site de ressources qui vulgarise et
-  valorise les données environnementales de l'ADEME.
+- **[Mes Aides Vélo](https://mesaidesvelo.fr)** trouve les aides pour acheter
+  un vélo dans votre ville, ainsi que d'autres aides pour la mobilité (prime
+  conversion voiture, forfait mobilités durables).
+- **[Futureco](https://futur.eco)** propose des simulateurs d'empreinte climat
+  d'un trajet en avion, en ferry, mais aussi un calculateur de coût en € d'un
+  trajet en voiture.
+- **[Impact CO2](https://impactco2.fr)** offre des ressources diverses pour
+  vulgariser les données environnementales de l'ADEME.
+- **[Ekofest](https://ekofest.fr)** aide les organisateurs de festival et
+  autres événements à calculer l'empreinte carbone complète de leur événement.
```

Comment:
```suggestion
- **[Ekofest](https://ekofest.fr)** est un simulateur d'empreinte carbone pour l'organisation d'événements festifs et culturels éco-responsables.
```

Reply:
Je sais pas ce que tu en penses @Clemog ? C'est surtout le "organisateurs" et "complète" qui me dérange dans la proposition de Mael.

Reply:
@EmileRolley mieux vaut garder le style présent "[verbe] les [utilisateurs] " pour bien sonner avec les autres exemples, plutôt que "est un xxx". Pour le reste, à vous de voir :)

Reply:
Je vous laisse merger une fois décidé

Reply:
J'ai fait une nouvelle proposition, on peut encore itérer :)